src/pipeline.cpp

```diff
@@ -743,14 +735,14 @@ PIC.addClassToPassName(decltype(CREATE_PASS)::name(), NAME);
     }
 }
 
-NewPM::NewPM(std::unique_ptr<TargetMachine> TM, OptimizationLevel O, OptimizationOptions options) :
+NewPM::NewPM(std::unique_ptr<TargetMachine> TM, OptimizationLevel O, LLVMContext &ctx, OptimizationOptions options) :
     TM(std::move(TM)),
 #if JL_LLVM_VERSION < 160000
     SI(false),
-    PIC(createPIC(SI)),
 #else
-    PIC(createPIC()),
+    SI(ctx, false),
```

Comment:
While you're at it might as well refactor to take the FAM as well, for the cfg verification.